pkg/image/util/resolve.go

```diff
@@ -0,0 +1,78 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openshift/origin/pkg/client"
+	imageapi "github.com/openshift/origin/pkg/image/api"
+)
+
+// ResolveImagePullSpec resolves the provided source which can be "docker", "istag" or
+// "isimage" and returns the full Docker pull spec.
+func ResolveImagePullSpec(client *client.Client, source, name, defaultNamespace string) (string, error) {
```

Comment:
@soltysh there are tests for the command itself already, I can add unit tests but they will basically be verifying the same behavior.